orcid_service/app.py

```diff
@@ -1,35 +1,22 @@
-
-import logging.config
-from flask import Flask
+from werkzeug.serving import run_simple
+from adsmutils import ADSFlask
 from views import bp
-from flask.ext.consulate import Consul, ConsulConnectionError
 from flask.ext.discoverer import Discoverer
-from flask.ext.sqlalchemy import SQLAlchemy
-from .models import db
 
-def create_app(config=None):
-    app = Flask(__name__, static_folder=None)
-    app.url_map.strict_slashes = False
+def create_app(**config):
+
+    if config:
```

Comment:
We could use this one line approach too: app = ADSFlask(__name__, static_folder=None, local_config=config or {})